addons/hr_holidays/models/hr_leave_accrual_plan_level.py

```diff
@@ -12,16 +12,26 @@
 
 DAYS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat']
 MONTHS = ['jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
+# Used for displaying the days and reversing selection -> integer
+DAY_SELECT_VALUES = [str(i) for i in range(1, 29)] + ['last']
+# Integer to ordinal string, found of stackoverflow: https://stackoverflow.com/a/50992575
+DAY_SELECT_DISPLAYS = ["%d%s" % (i, "tsnrhtdd" [(i / 10 % 10 != 1) * (i % 10 < 4) * i % 10::4]) for i in range(1, 29)] + ['last day']
```

Comment:
If I understand well, purpose is to make the match between selection key (1, 2, ...) and its ordinal value. Issue is that it will probably not be translated and probably not match user language preferences.

Reply:
Arf I completely forgot about translations and I didn't want to copy paste a 28~ line array for each fields below.
However I don't see why it wouldn't translate well? Any insight on this?

Reply:
I know people who master translations and language management for dates, isn't it @mart-e  ?

Reply:
```py
>>> num2words(57, lang='en', to='ordinal_num')
57th
```
maybe ?

Reply:
I will try it out, thanks a lot! 😃